x/file_test.go

```diff
@@ -95,7 +91,12 @@ func TestIsMissingOrEmptyDir(t *testing.T) {
 	output = IsMissingOrEmptyDir("./doesnotexist")
 	require.NotEqual(t, nil, output)
 
-	if err := os.RemoveAll(testFilesDir); err != nil {
+	defer deleteDirs(t, testFilesDir)
```

Comment:
See above comment regarding relocating the defer calls

Reply:
Hii @matthewmcneely, done